#161342793 Test application functionality

victor-nweze-deel opened the pull request:
#### What does this PR do?
- Add tests for application functionality

#### Description of Task to be completed?
- Install tests suites
- Write tests
- Update travis file

#### How should this be manually tested?
- Git clone this repo
- Cd into the `haven-ah-frontend` folder
- Run `npm test`

#### What are the relevant pivotal tracker stories?
[#161342793](https://www.pivotaltracker.com/story/show/161342793)

#### Screenshots (if available)
<img width="907" alt="screen shot 2018-12-05 at 8 24 15 am" src="https://user-images.githubusercontent.com/41582565/49498140-5faa0580-f86a-11e8-90eb-8fc224d9e2fc.png">
<img width="1168" alt="screen shot 2018-12-05 at 7 47 29 am" src="https://user-images.githubusercontent.com/41582565/49498144-62a4f600-f86a-11e8-8706-9d6ea32dc694.png">

victor-nweze-deel committed:
ch(integration): Integrate Travis CI with application

- Install jest
- Write dummy test
- Add a travis file

TheoOkafor reviewed (commented):
Nice work @ProfJigsaw

ucheg6 committed:
ch(test): Write tests for app functionality

- Install test suite packages
- Write tests
- Update travis file to pipe coverage
- Add tests scripts for end-to-end testing

PriscillaSam commented:
Great work here @ProfJigsaw and @ucheg6. 👍

seyi-adeleke reviewed (approved):
🔥
